docs/develop/integrations/netlify-integration.md

```diff
@@ -31,9 +31,11 @@ After pressing the `Create` button you'll see the new `chat` cache in the list o
 
 ![Chat cache](/img/console-caches-chat.png)
 
-Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure that you choose the same region when you generate a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, choose the cloud provider and region you used to create your cache, choose the `Super User Token` token type, and hit the `Generate Token` button. 
+Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure that you choose the same region when generating a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, and choose the cloud provider and region you used to create your cache. Since the cache is already created, we will use a fine-grained token that will allow the worker to read from and write to the cache; but will not allow it to do control plane operations such as delete. This is especially helpful if you want to manage the security of control plane and data plane operations separately.
 
-![Generate token](/img/console-generate-token-form.png)
+Choose the `Fine-Grained Access Token` token type, select `chat` as `Cache Name` from the drop down, and `readwrite` as `Role Type`. The `Super User Token` is used for managing control plane operations. More information about Momento authentication can be found [here](https://docs.momentohq.com/develop/basics/working-with-momento-auth-tokens). Hit the `Generate Token` button.
```

Comment:
This app isn't reading or writing to cache. It's 100% pubsub and the cache has already been created, so no need to discuss control or data plane in this context really. These instructions need to be updated to describe how to generate a `publishsubscribe` topics token for all topics in the `chat` cache.

Reply:
Scratch that :-) These chat apps require a superuser token because they generate expiring FGAC tokens to the browsers. The docs are correct as written originally.

Reply:
Done